components/badge/badge.css

```diff
@@ -0,0 +1,57 @@
+.badge-icon-group{
+    padding: 1rem;
+    height: 10vh;    
+}
+
+
+.badge{
+    position: relative;
+}
+
+.badge-icon{
+    position: absolute;
+    border-radius: 50%;
+    background-color: var(--white);
+}
+
+.badge-icon-sm{
+    color: var(--green);
+    right: 1.125rem;
+    bottom: 0.75rem;
+    font-size: 1rem;
+    
+}
+
+.badge-icon-xl{
+    color: var(--red);
+    right: 1.25rem;
+    bottom: 1rem;
+   
+}
+
```

Comment:
Remove this all extra whitespace(Format document)

Reply:
Done